### DISCUSSION TITLE
Cursor disappears in insert mode with Neovim

### DISCUSSION THREAD

**Discussion by maxcountryman** [arcticicestudio/nord#206] [Help & Support]
I'm not quite sure what's going on here...when I first setup Neovim, insert mode was working fine (i.e. it showed a skinny version of the cursor marker). After I setup my theme the cursor completely disappeared when in insert mode.

I'm using macOS 10.14.5, TMUX 2.8, NVIM v0.3.8, and this package to install and setup Nord.

Any help is greatly appreciated--this is surprisingly annoying!

**Comment by arcticicestudio** [comment]
> Related to nordtheme/vim#72 

Hi @maxcountryman, I guess this is related to your _tmux_ setup. There are several issues like nordtheme/vim#72 where _tmux_ is not correctly configured for the running environment and terminal emulator.
Could you please follow the instructions in https://github.com/arcticicestudio/nord-tmux/issues/15#issuecomment-341715220 to see if it fixes you problem?

**Comment by arcticicestudio** [comment]
@maxcountryman Any update on the problem? have you been able to resolve it using the instructions from the linked comment?

**Comment by maxcountryman** [comment]
Nope—this continues to be an issue.

**Comment by svengreb** [comment]
Thank you for your patience! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance).

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“. Read the [announcement about reaching the “clean“ contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord‘s discussions for more details about the goal.

Due to the scope of this issue of not being “purely technical“, or being related to the project‘s code at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.

@maxcountryman Since there has been some time without further communication, it would be great if you could quickly give some feedback if the problem has been “resolved“ in the meantime. Maybe this was related to NeoVim itself or maybe you found a solution that could also help others.
